src/backend/storeManagers/gog/games.ts

```diff
@@ -989,7 +989,9 @@ export async function updateGOGPlaytime(
     return
   }
 
-  const response = await postPlaytimeSession({ ...data, appName })
+  const response = await postPlaytimeSession({ ...data, appName }).catch(
+    () => null
```

Comment:
maybe we can log that something failed here? or re-try posting the playtime?

Reply:
Below that, if the response fails for whatever reason we add it to retry queue, which will be ran on next Heroic startup. Isn't that pretty much that? What else do you suggest?

Reply:
oh I see, what confused me was that the rest of the code was already in the file and this catch is new, so I imagined it was doing something extra, so this is to make response `null` when something fails so `!response` in the next line would be true?

I understand before it was throwing an exception instead of returning null?

Reply:
> I understand before it was throwing an exception instead of returning null?

Technically `postPlaytimeSession` shouldn't throw any exceptions, however if we were ever to change how the function behaves we have this another layer of safety